LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+Source code in this repository is variously licensed under the Apache License
```

Comment:
Ugh, I forgot that beats used a dual license. We want the plain Apache license: https://github.com/elastic/elastic-agent-libs/blob/main/LICENSE

Sorry for the wrong suggestion.